pkg/infra/kvstore/kvstore.go

```diff
@@ -58,3 +59,7 @@ func (kv *NamespacedKVStore) Del(ctx context.Context, key string) error {
 func (kv *NamespacedKVStore) Keys(ctx context.Context, keyPrefix string) ([]Key, error) {
 	return kv.kvStore.Keys(ctx, kv.orgId, kv.namespace, keyPrefix)
 }
+
+func (kv *NamespacedKVStore) GetAll(ctx context.Context) (map[int64]map[string]string, error) {
```

Comment:
// GetAll returns all the keys and values stored per organization. It returns a map of org -> key -> value.